chirp/drivers/ga510.py

```diff
@@ -452,8 +624,13 @@ def set_memory(self, mem):
 
         _nam.name = mem.name.ljust(10)
 
+        if self.MODEL == "GS-5B" or self.MODEL == "SHX-8800" or self.MODEL == "AC-580" or self.MODEL == "XTR-5":
```

Comment:
It would be better to set one flag on the SHX-8800 common parent of `has_allow_tx` or something, set that in one place and check just that flag here, instead of listing all the models. You could also do this:
```
if isinstance(self, Senhaix8800Radio):
```